docs/documentation/cl_options/index.rst

```diff
@@ -0,0 +1,40 @@
+Command Line Arguments
+======================
+
+What starting vimiv form the command line you have the ability to pass a number of
+different argument to vimiv. The general calling structure is depicted below:
+
+.. include:: synopsis.rstsrc
+
```

Comment:
I would put the examples in front of the detailed list. As a reader the examples are more interesting, when looking for a reference you will most likely click on it directly and not "discover" it.

Reply:
rebased and fixed in e6a562414448933601ee2b9b8f0000c9fe266fc2